Description: The irssi plugin sends a "Happy New Year" message to all
 open conversations without asking the user.
 The patch disables this behavior by default.
Author: Felix Geyer <fgeyer@debian.org>
Bug-Debian: http://bugs.debian.org/567751
Bug-Ubuntu: https://bugs.launchpad.net/bugs/502948

Index: purple-plugin-pack-2.6.2/irssi/irssi.c
===================================================================
--- purple-plugin-pack-2.6.2.orig/irssi/irssi.c	2010-02-03 19:39:55.000000000 +0100
+++ purple-plugin-pack-2.6.2/irssi/irssi.c	2010-02-03 19:40:08.000000000 +0100
@@ -139,7 +139,7 @@
 	purple_prefs_add_none(PREFS_ROOT);
 	purple_prefs_add_bool(TEXTFMT_PREF, TRUE);
 	purple_prefs_add_bool(DATECHANGE_PREF, TRUE);
-	purple_prefs_add_bool(SENDNEWYEAR_PREF, TRUE);
+	purple_prefs_add_bool(SENDNEWYEAR_PREF, FALSE);
 }
 
 PURPLE_INIT_PLUGIN(PLUGIN_STATIC_NAME, irssi_init, irssi_info)
